src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -174,11 +174,18 @@ public static CodeWriter.CodeWriterScope WriteMethodDeclaration(this CodeWriter
 
         public static CodeWriter WriteMethodDocumentation(this CodeWriter writer, MethodSignatureBase methodBase)
         {
-            writer.WriteXmlDocumentationSummary($"{methodBase.Description}");
+            writer.WriteMethodDocumentationBase(methodBase);
+            writer.WriteXmlDocumentation("remarks", $"{methodBase.DescriptionText}");
```

Comment:
Why not move `writer.WriteXmlDocumentation("remarks", $"{methodBase.DescriptionText}");` into `WriteMethodDocumentationBase`?

Reply:
wrong commit